hips/tiles/tile_meta.py

```diff
@@ -17,14 +23,26 @@ class HipsTileMeta:
         HEALPix pixel number
     file_format : {'fits', 'jpg', 'png'}
         File format
+    frame : {'icrs', 'galactic', 'ecliptic'}
+        Sky coordinate frame
     tile_width : `int`
         Tile width (in pixels)
+
+    Examples
+    --------
+    >>> from hips.tiles import HipsTileMeta
+    >>> tile_meta = HipsTileMeta(order=3, ipix=450, file_format='fits', frame='galactic', tile_width=512)
+    >>> tile_meta.skycoord_corners
+    <SkyCoord (Galactic): (l, b) in deg
+    [( 264.375, -24.62431835), ( 258.75 , -30.        ),
+     ( 264.375, -35.68533471), ( 270.   , -30.        )]>
     """
 
-    def __init__(self, order: int, ipix: int, file_format: str, tile_width: int = 512) -> None:
+    def __init__(self, order: int, ipix: int, file_format: str, frame: str = 'galactic', tile_width: int = 512) -> None:
```

Comment:
I suggest to change the default to "icrs". That's more common for HiPS data, and it's also the default for `SkyCoord`.